### PR TITLE
Remove clipboard-read permission

### LINKED ISSUE #101: Is "clipboard" permission replaced by "clipboard-read" and "clipboard-write"?
[The Permissions spec](https://w3c.github.io/permissions/#dom-permissionname-clipboard) still mentions `"clipboard"` but it seems it's removed from the Clipboard spec. Should the Permissons spec be updated to use the new permission names?

### PR DESCRIPTION
**Pull request by saschanaz**
Closes #101, see also https://github.com/w3c/permissions/issues/175

For normative changes, the following tasks have been completed:

 * [ ] Modified Web platform tests (link to pull request)

Implementation commitment:

 * [x] WebKit - Never shipped, and doesn't support this change.
 * [ ] Chromium (https://bugs.chromium.org/p/chromium/issues/detail?id=)
 * [x] Gecko - Never shipped, and doesn't support this change.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/clipboard-apis/pull/132.html" title="Last updated on Oct 26, 2020, 5:05 AM UTC (1084f2e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/clipboard-apis/132/4231c62...1084f2e.html" title="Last updated on Oct 26, 2020, 5:05 AM UTC (1084f2e)">Diff</a>

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
Thanks for doing this work @saschanaz.

**Comment by pwnall** [comment]
Chrome has shipped the `clipboard-read` permission, so this would be a breaking change for us.

@dway123 do we have any data we could share about the magnitude of the breakage?

**Comment by dway123** [comment]
Sure. The suggested removal of `clipboard-read` here implies [requiring user activation](https://github.com/w3c/clipboard-apis/issues/52) to gate Clipboard read. Therefore, this should break any site that currently doesn't require user activation to read the clipboard.

While we don't know the exact magnitude of the breakage (we don't currently have metrics on clipboard read use without user activation), we do know that:
- .03% of pageloads currently [use](https://chromestatus.com/metrics/feature/timeline/popularity/2369) `clipboard.read()`, and .22% of pageloads currently [use](https://chromestatus.com/metrics/feature/timeline/popularity/2371) `clipboard.readText()`. The subset of these sites that doesn't do so requiring user activation would be broken by this change.
- As of September 2020, 11,088 unique sites made use of the Async Clipboard API, according to the HTTP Archive. This number may be higher now, but also includes sites using clipboard write but not clipboard read, or clipboard read, but with user activation, so may represent more than the complete list of affected sites.
- Remote desktop applications, [including](https://github.com/w3c/clipboard-apis/issues/75) cendio.com, as well as Chrome Remote Desktop, would be broken by this change, as this would disallow them to propagate clipboard updates.

**Comment by rniwa** [comment]
> Sure. The suggested removal of `clipboard-read` here implies [requiring user activation](https://github.com/w3c/clipboard-apis/issues/52) to gate Clipboard read. Therefore, this should break any site that currently doesn't require user activation to read the clipboard.

Aren't those sites also broken in Firefox & Safari? Or are you more concerned about Chrome only content?

**Comment by dway123** [comment]
> Aren't those sites also broken in Firefox & Safari? Or are you more concerned about Chrome only content?

I suspect but have not verified these sites are also broken in Firefox and Safari, if they require user gesture for all Clipboard reads. I only discussed Chrome content because the question was regarding breakage in Chrome, and I only had public use information for Chrome, like [chromestatus feature use data](https://chromestatus.com/metrics/feature/timeline/popularity/2369). Please share information about breakage in Firefox & Safari, if possible, to provide more color for this conversation. Sorry, I'm not well acquainted with available public data for Firefox & Safari.

**Comment by marcoscaceres** [comment]
> Please share information about breakage in Firefox & Safari, if possible, to provide more color for this conversation.

I don't think Firefox typically collects this kind of information (though some from Mozilla can correct me). They would need to check specific sites to see if they are breaking. However, likelihood is that they are not broken if the read is happening along with a user gesture... hopefully sites have figured that out already and at mitigating breakage by calling the API as a result of user activation.

**Comment by saschanaz** [comment]
I can confirm that Mozilla doesn't have a use counter for `navigator.clipboard`.

**Comment by marcoscaceres** [comment]
So, just chiming in a chair here. The appropriate course of action would be to merge this to match implementations. Then, as a followup, if folks from Google would still like to pursue this option, it might be worth sending a PR reverting the changes made in this PR.

**Comment by rniwa** [comment]
> > Aren't those sites also broken in Firefox & Safari? Or are you more concerned about Chrome only content?
> 
> I suspect but have not verified these sites are also broken in Firefox and Safari, if they require user gesture for all Clipboard reads. I only discussed Chrome content because the question was regarding breakage in Chrome, and I only had public use information for Chrome, like [chromestatus feature use data](https://chromestatus.com/metrics/feature/timeline/popularity/2369). Please share information about breakage in Firefox & Safari, if possible, to provide more color for this conversation. Sorry, I'm not well acquainted with available public data for Firefox & Safari.

Safari does not collect such data either. Given Safari always requires user gesture to paste, it's pretty safe to assume such websites are broken in Safari. If those websites are broken in Safari, on the other hand, it means the website has some Chrome/Chromium specific behavior. Even in such a scenario, this will likely remain Chrome/Chromium only permissions so the two-implementation requirement means that this permission should not be added back to the spec.

**Comment by marcoscaceres** [comment]
Ok, I'm going to go ahead and merge this, as the two engine requirement hasn't been met for this feature.

File: index.bs [hunk 1]
```diff
@@ -960,87 +959,41 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 	There are 4 clipboard permissions:
```
Comment (yume-chan) [inline index.bs:960]: It still says `4` instead of `2`

**Pull request merged**